.github/actions/auto-major-minor-release-v1/action.yml

```diff
@@ -0,0 +1,168 @@
+        DEV_HASH="$(git rev-parse "origin/$DEFAULT_BRANCH")"
+        COMMITS="$(git log --oneline --no-merges --graph --abbrev-commit $MASTER_HASH..$DEV_HASH)"
+        echo "Commits:"
+        echo "$COMMITS"
+
+        echo "commits<<AUTO_RELEASE_HEREDOC" >> "$GITHUB_OUTPUT"
+        echo "$COMMITS" >> "$GITHUB_OUTPUT"
+        echo "AUTO_RELEASE_HEREDOC" >> "$GITHUB_OUTPUT"
+      env:
+        RELEASE_BRANCH: ${{ inputs.release-branch }}
+        DEFAULT_BRANCH: ${{ inputs.default-branch }}
+
+    - name: Create Docs issue
+      uses: dacbd/create-issue-action@ba4d1c45cccf9c483f2720cefb40e437f0ee6f7d
+      id: new-docs-issue
+      with:
+        token: ${{ inputs.project_token }}
+        repo: docs-devtools-html
+        assignees: enlarsen
+        title: "[TEST] ${{ github.event.repository.name }} v${{ steps.get-new-version.outputs.version }} Release Notes Needed"
+        labels: release
+        body: |
+          Please create release notes for ${{ github.event.repository.name }}. Here are all changes since last release
+
+          The following commit(s) were added in this release:
+          ${{ steps.commit-list.outputs.commits }}
+
+
+    - name: Create QA issue
+      uses: dacbd/create-issue-action@ba4d1c45cccf9c483f2720cefb40e437f0ee6f7d
+      id: new-issue
+      with:
+        token: ${{ inputs.token }}
+        title: "[TEST] ${{ github.event.repository.name }} v${{ steps.get-new-version.outputs.version }} QA Needed"
+        labels: release
+        body: |
+          The develop branch of ${{ github.event.repository.name }} is ready for QA.
+
+          Please add comments in this Issue to state any bugs found and tag a developer. Once ${{ github.event.repository.name }} passes QA, please attach the report to the api-team-releases slack channel, close this Issue and move this Issue to the Done column.
+
+          The following commit(s) were added in this release:
+          ${{ steps.commit-list.outputs.commits }}
+    - name: Store issue URL
+      id: issue-url
+      shell: bash
+      run: |
+        url="$(echo $json | jq .html_url | sed 's/"//g' )"
+        echo "url=$url" >> $GITHUB_OUTPUT
+      env:
+        json: ${{  steps.new-issue.outputs.json  }}
+    - name: Save issue-id
+      id: issue-id
+      shell: bash
+      run: |
+        issueId="$(echo $json | jq -r .node_id)"
+        echo "id=$issueId" >> "$GITHUB_OUTPUT"
+      env:
+        json: ${{  steps.new-issue.outputs.json  }}
+    - name: Move to board
+      uses: dequelabs/axe-api-team-public/.github/actions/add-to-board-v1@main
+      with:
+        token: ${{ inputs.project_token }}
+        issue-id: ${{ steps.issue-id.outputs.id }}
+        # 66 taken from the url: https://github.com/orgs/dequelabs/projects/66
+        project-number: 66
+    - name: Slack Notification
+      uses: rtCamp/action-slack-notify@12e36fc18b0689399306c2e0b3e0f2978b7f1ee7
+      env:
+        SLACK_CHANNEL: api-team-releases
```

Comment:
Does it matter if this channel name is made public?

Reply:
Its public in our other release action, so I hope not